src/explorer/model/MavenProject.ts

```diff
@@ -51,7 +51,8 @@ export class MavenProject implements ITreeItem {
     }
 
     public get groupId(): string {
-        return this._pom?.project?.groupId?.[0] ?? this.parent?.groupId;
+        return this._pom?.project?.groupId?.[0] ?? this._pom?.project?.parent?.[0]?.groupId?.[0];
+        //this.parent?.groupId;
```

Comment:
nit: remove?

Reply:
I'd prefer to append `?? this.parent?.groupId` althrough it may not be necessary...